src/main/java/com/softserve/sportshub/comment/controller/CommentController.java

```diff
@@ -0,0 +1,61 @@
+package com.softserve.sportshub.comment.controller;
+
+import com.softserve.sportshub.comment.dto.CreateCommentDto;
+import com.softserve.sportshub.comment.dto.ReadCommentDto;
+import com.softserve.sportshub.comment.service.CommentService;
+import com.softserve.sportshub.user.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.Principal;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class CommentController {
+
+
+    public CommentService commentService;
+    public UserService userService;
+
+    public CommentController(CommentService commentService, UserService userService) {
+        this.commentService = commentService;
+        this.userService = userService;
+    }
+
+    @GetMapping("/comments")
+    public List<ReadCommentDto> getAllComments() {
+        return commentService.list();
+    }
+
+    @PostMapping("/comment")
+    public ResponseEntity<ReadCommentDto> createComment(Principal principal, @RequestBody CreateCommentDto createCommentDto) {
+        String username = principal.getName();
+        ReadCommentDto savedComment = commentService.save(createCommentDto, username);
+        return ResponseEntity.ok().body(savedComment);
+    }
+
+    @DeleteMapping("/comment/{id}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public void deleteComment(@PathVariable Long id) {
+        commentService.delete(id);
+    }
+
+
+    @PostMapping("/{id}/like")
```

Comment:
here you're missing a /comment i believe, now its /api/{id}/like to like a comment